### PR TITLE
team building for team context

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] Your results show special case bottomRow
- [x] Pass `namespace` as a param to `userInterestingPeopleRpcPromise`
- [x] Make userInterestingPeopleRpcPromise filter hellobot/you when namespace === `teams` (@hyperobject )

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/team-building/search-result/you-result.tsx [hunk 1]
```diff
@@ -12,14 +12,20 @@ const YouResult = React.memo((props: ResultProps) => {
...
+    bottomRow = (
+      <Kb.Text type="BodySmall">
+        {props.isPreExistingTeamMember ? 'Already in team' : 'Add yourself to the team'}
+      </Kb.Text>
+    )
+  } else {
+    bottomRow = <Kb.Text type="BodySmall">Write secure notes to yourself</Kb.Text>
+  }
+
+  return <CommonResult {...props} onAdd={onSelfChat} rowStyle={styles.rowContainer} bottomRow={bottomRow} />
```
Comment (buoyad) [inline shared/team-building/search-result/you-result.tsx]: I think `onAdd` should have the regular behavior outside of the chat context, rn it always takes you to your self chat

Comment (chrisnojima) [reply shared/team-building/search-result/you-result.tsx]: fixed. also fixed clicking on add in wallets which calls cancel twice and closes the modal!

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/team-building/search-result/user-result.tsx [hunk 2]
```diff
@@ -15,15 +15,26 @@ const UserResult = React.memo((props: ResultProps) => {
       rowStyle={styles.rowContainer}
       rightButtons={
         !props.isPreExistingTeamMember && (
-          <ActionButton inTeam={props.inTeam} onAdd={props.onAdd} onRemove={props.onRemove} />
+          <ActionButton
+            inTeam={props.inTeam}
+            onAdd={(e: React.BaseSyntheticEvent) => {
+              e.stopPropagation()
+              props?.onAdd()
```
Comment (jakob223) [inline shared/team-building/search-result/user-result.tsx]: I don't understand what this is protecting against.

Comment (chrisnojima) [reply shared/team-building/search-result/user-result.tsx]: Onadd was always optional. Since we're doing 2 things we need to check

**Pull request merged**